packages/assets/assets/alert-circle.svg

```diff
@@ -0,0 +1,12 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you please confirm that this icon is not duplicated?

Reply:
yes it's not. the one that are there are filled
![image](https://github.com/user-attachments/assets/5a738b8f-504f-4e02-bf42-beb3acc8746e)
![image](https://github.com/user-attachments/assets/b0911775-6bde-42d1-8ea7-f24415e515ce)

this is what we need:
![image](https://github.com/user-attachments/assets/98ef374c-ff83-432b-86c9-c23a55bd2862)
it has a stroke not fill